src/constants/index.ts

```diff
@@ -1,77 +0,0 @@
-// Copyright (c) 2022 Uber Technologies, Inc.
```

Comment:
Looks like this file should be moved to ./src, but it is deleted

Reply:
It has a lot of changes in it, so git decided that index.ts in src folder is taken form src/layers/types.ts